apps/etterlatte-beregning/src/main/kotlin/sanksjon/SanksjonRepository.kt

```diff
@@ -100,8 +102,15 @@ data class Sanksjon(
     val sakId: Long,
     val fom: YearMonth,
     val tom: YearMonth?,
-    val saksbehandler: String,
-    val opprettet: Tidspunkt,
-    val endret: Tidspunkt,
```

Comment:
Jeg vet ikke om det er lurt å evt. ha med et timestamp her også, men dette er nok mye viktigere data å ta vare på uansett :p

Reply:
Litt usikker på om jeg forstår deg her, men tanken min her var å bruke kilde for både opprettet og endret slik at man får opp saksbehandler og tidspunkt på begge

Reply:
Ah, denne har jo selvfølgelig med tidspunkt. Never mind me 😅